### DISCUSSION TITLE
REGISTRATION METHOD WITH IBAX NETWORK TWITTER IS NOT SAFE!!!

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2052] [Bug Submission]
Records management with Twitter does not represent the actual number of users. because hundreds of fake accounts can be created on Twitter.

for example my email address: umity7965.@gmail.com

Let me give an example from this e-mail address.

Thousands of accounts can be created on the same e-mail address by creating different combinations such as "+" "-" on the same e-mail address.

For example;
1) umity7965+@gmail.com
2)umity7965+-@gmail.com
3)umity7965++-@gmail.com
4)umity7965++++-@gmail.com

and the account can be created this way forever.

As a result, people can loot coins as referral earnings in future airdrops or referral campaigns.

![IMG_20221005_025316](https://user-images.githubusercontent.com/96292241/193952108-473dc0bc-b9e6-4f84-a002-9c6793e70623.jpg)

![IMG_20221005_025303](https://user-images.githubusercontent.com/96292241/193952124-9a93a49d-d135-431d-a042-eca1ea7bd074.jpg)

**Comment by yolandadadada** [comment]
Officials will judge the authenticity of the account, and we reject bot behavior.

**Comment by MrDino3813** [comment]
You are right alot of can successfully game the system even if the Ibax time manages to identify some bot behavior.

My solution and suggestions is Ibax should integrate BrightID as a verify method to authenticate I avatar in their server

**Reply by Akrepkapo** [reply]
Yes
